unit_tests/CMakeLists.txt

```diff
@@ -75,7 +75,7 @@ set(
   with_durability
   json_apis_from_client
   zone_report
-  ticket_administration
```

Comment:
Are we removing this test?

Reply:
That was probably my mistake.  I will add back in.